src/rules/prefer-in-document.js

```diff
@@ -20,7 +20,10 @@ export const meta = {
   fixable: "code",
   messages: {
     "use-document": `Prefer .toBeInTheDocument() for asserting DOM node existence`,
+    "invalid-combination-length-1": `Invalid combination of {{query}} and .toHaveLength(1). Did you mean to use {{allQuery}}?`,
+    "replace-query-with-all": `Replace {{query}} with {{allQuery}}`,
```

Comment:
nit: from what I've seen its common practice to have spaces inbetween these:

```suggestion
    "invalid-combination-length-1": `Invalid combination of {{ query }} and .toHaveLength(1). Did you mean to use {{ allQuery }}?`,
    "replace-query-with-all": `Replace {{ query }} with {{ allQuery }}`,
```